packages/explat-client/src/test/create-explat-client.ts

```diff
@@ -88,9 +88,9 @@ describe( 'ExPlatClient.loadExperimentAssignment single-use', () => {
 		await expect(
 			client.loadExperimentAssignment( validExperimentAssignment.experimentName )
 		).resolves.toEqual( validExperimentAssignment );
-		expect( ( mockedConfig.logError as MockedFunction ).mock.calls ).toMatchInlineSnapshot(
-			`Array []`
-		);
+		// expect( ( mockedConfig.logError as MockedFunction ).mock.calls ).toMatchInlineSnapshot(
+		// 	`Array []`
+		// );
```

Comment:
This is not a good use case for snapshots 🙂 If we want to check that a mock function was not called, there is a helper exactly for that:
```js
expect( mockedConfig.logError ).not.toHaveBeenCalled();
```
Jest will issue more sensible error messages when the intent is made clear like this.